pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${bdg-formats.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.bdgenomics.qc-metrics</groupId>
+      <artifactId>qc-metrics-core</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
I'm still futzing with IntelliJ and trying to get it to see this dependency; this doesn't have to block this PR but it is a little strange that we're seeing such not-understood behavior.